zips/zip-0227.rst

```diff
@@ -369,9 +381,15 @@ For every transaction:
 
   - It MUST be the case that $0 < \mathtt{assetDescSize} \leq 512$.
   - It MUST be the case that $\mathsf{asset\_desc}$ is a string of length $\mathtt{assetDescSize}$ bytes.
-  - Elements of every Issue Note in ``IssueAction`` MUST be valid encodings of the types given in the `Issue Note`_ section, and MUST encode the same $\mathsf{AssetBase}$. 
+  - Every Issue Note in ``IssueAction`` MUST be a valid encoding of the $\mathsf{Note^{Issue}}$ type, and MUST encode the same $\mathsf{AssetBase}$. 
   - This $\mathsf{AssetBase}$ MUST satisfy the derivation from the issuance validating key and asset description described in the `Specification: Asset Identifier`_ section.
   - It MUST be the case that $\mathsf{issued\_assets}_{\mathsf{OUT}}(\mathsf{AssetBase}).\mathsf{final} \neq 1$.
+  - If $\mathsf{issued\_assets}_{\mathsf{OUT}}(\mathsf{AssetBase}).\mathsf{ref\_note} = \bot$, then let $\mathsf{note}_1$ be the first Issue Note in the Issuance Action.
+
+      - The recipient address $(\mathsf{d}, \mathsf{pk}_{\mathsf{d}})$ of $\mathsf{note}_1$ MUST be the default diversified payment address derived from the all-zero Orchard spending key, as described in the `Reference Notes`_ section.
+      - The value $\mathsf{v}_0$ of $\mathsf{note}_1$ MUST be $0$, and the $\text{ρ}$ field of $\mathsf{note}_1$ MUST have been computed as described in the `Computation of ρ`_ section.
```

Comment:
> and the $\text{ρ}$ field of $\mathsf{note}_1$ MUST have been computed as described in the `Computation of ρ`_ section.

We have this check done for all notes later, so it is kind of redundant. But I thought it should be added before the node updates the global state in the next line.

Reply:
Removed the redundant check